src/Components/NotesCard/NotesCard.jsx

```diff
@@ -23,38 +28,54 @@ const NotesCard = ({
         <div className="card-body">
           <p className="card-description">{notesBody}</p>
         </div>
+
         {showIcons && (
           <div className="card-btns flex">
-            <span
-              className="material-icons-outlined"
-              onClick={() => editHandler({ ...notes, _id: _id })}>
-              edit
-            </span>
-            <span className="material-icons">palette</span>
-            <span className="material-icons">label</span>
-
-            <span
-              className="material-icons"
-              onClick={
-                path !== "/archive"
-                  ? () => deleteNoteHandler(_id)
-                  : () => archiveDeleteHandler(_id)
-              }>
-              delete
-            </span>
+            {path !== "/trash" ? (
+              <>
+                <span
+                  className="material-icons-outlined"
+                  onClick={() => editHandler({ ...notes, _id: _id })}>
+                  edit
+                </span>
 
-            {path !== "/archive" ? (
-              <span
-                className="material-icons"
-                onClick={() => addArchiveHandler(notes)}>
-                archive
-              </span>
+                <span className="material-icons">label</span>
+                {path !== "/archive" ? (
+                  <span
+                    className="material-icons"
+                    onClick={() => addArchiveHandler(notes)}>
+                    archive
+                  </span>
```

Comment:
You could just create separate cards for trash and archive, the code will looks more cleaner. (your preference!)